ts/state/ducks/lightbox.ts

```diff
@@ -278,12 +292,8 @@ function showLightbox(opts: {
     const attachments = filterValidAttachments(message.attributes);
     const loop = isGIF(attachments);
 
-    const authorId =
-      window.ConversationController.lookupOrCreate({
-        serviceId: message.get('sourceServiceId'),
-        e164: message.get('source'),
-        reason: 'conversation_view.showLightBox',
-      })?.id || message.get('conversationId');
+    const authorId = getAuthorId(message);
```

Comment:
I had the problem here that the conversationId gets set as the authorId, the conversation id only gets set as fallback when "window.ConversationController.lookupOrCreate" is null/undefined.

But for this functionality I always need the conversationId, so I introduced a new variable "authorId" and replaced all previous usages with that one.

Is it intended that the variable got set like this in the first place and is there a gotcha that I haven't seen?